server/main.py

```diff
@@ -23,7 +23,10 @@
 
 def generate_random_string(length=8):
     letters = string.ascii_lowercase
```

Comment:
First of all why is the length=8? This means only 26 to the power 8 keys can be generated?Which is about 200 billion, Ik we might not hit that scale anytime soon but by design this is flawed. And having an infinite loop in the system is always a bad idea. 
Secondly this should be a seperate PR

Reply:
For now hold on to this don't work on this. I will raise a proper bug for requesting modification for key generation to be modified.